samples/uploadDirectory.js

```diff
@@ -19,7 +19,7 @@
 //   description: Uploads full hierarchy of a local directory to a bucket.
 //   usage: node files.js upload-directory <bucketName> <directoryPath>
 
-async function main(bucketName, directoryPath) {
+function main(bucketName = 'my-bucket', directoryPath = '.') {
```

Comment:
Should these default to the old values just for consistency?

Reply:
I'm not seeing defaults before this change - I think we require these explicitly or it breaks.

Reply:
My mistake, more so meant the commented out examples.